crates/bevy_input/src/keyboard.rs

```diff
@@ -407,3 +412,18 @@ pub enum KeyCode {
     /// The `Cut` key.
     Cut,
 }
+
+/// The scan code of a [`KeyboardInput`](crate::keyboard::KeyboardInput).
+///
+/// ## Usage
+///
+/// It is used as the generic `T` value of an [`Input`](crate::Input) to create a `Res<Input<ScanCode>>`.
+/// The resource stores the numeration of the buttons of a keyboard and can be accessed inside of a system.
+/// Use [`KeyCode`](KeyCode) for more meaningful/readable code
```

Comment:
I don't think this captures the distinction quite correctly. ScanCodes are used for semantic values, KeyCodes correspond to the position on a keyboard. We should link to external reading here, and update `KeyCode`'s docs to link back to here.

Reply:
See for example https://wiki.libsdl.org/CategoryKeyboard

Reply:
I think you mixed up the two
> SDL_Scancode values are used to represent the physical location of a keyboard key on the keyboard.
> SDL_Keycode values are mapped to the current layout of the keyboard
(from the website you linked)
> scancode: ScanCode
>     Identifies the physical key pressed
> virtual_keycode: Option<VirtualKeyCode>
>     Identifies the semantic meaning of the key
https://docs.rs/winit/latest/winit/event/struct.KeyboardInput.html (`winit` docs, since Bevy gets `KeyboardInput` from it)

Reply:
done